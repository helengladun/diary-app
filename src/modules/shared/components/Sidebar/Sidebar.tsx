import React from 'react';

export const Sidebar = () =>  (
    <aside className="main-sidebar">
        <h1 className="main-title">Diary app</h1>
        <div className="main-subtitle">Comment with no sense</div>
    </aside>
);