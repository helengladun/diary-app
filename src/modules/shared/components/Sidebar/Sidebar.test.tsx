import React from 'react';
import {shallow} from 'enzyme';
import {Sidebar} from './Sidebar';

describe('Sidebar component', () => {
    const wrapper = shallow(<Sidebar/>);

    it('make snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });

});