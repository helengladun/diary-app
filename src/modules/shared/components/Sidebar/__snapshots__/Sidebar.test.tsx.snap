// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component make snapshot 1`] = `ShallowWrapper {}`;
